packages/accounts-base/accounts_server.js

```diff
@@ -1078,11 +1078,13 @@ Ap._generateStampedLoginToken = function () {
 function expirePasswordToken(accounts, oldestValidDate, tokenFilter, userId) {
   var userFilter = userId ? {_id: userId} : {};
 
-  accounts.users.update(_.extend(userFilter, tokenFilter, {
-    $or: [
-      { "services.password.reset.when": { $lt: oldestValidDate } },
-      { "services.password.reset.when": { $lt: +oldestValidDate } }
-    ]
+  accounts.users.update(_.extend(userFilter, 
+    {$and: [tokenFilter, {
+      $or: [
+        { "services.password.reset.when": { $lt: oldestValidDate } },
+        { "services.password.reset.when": { $lt: +oldestValidDate } }
+      ]
+    }]
```

Comment:
question, what's this `...` notation? I have not seen it before and a hard one to get an answer for from google. why does it solve my "unknown operator" issue?